### PR TITLE
FEATURE: Show property with additional costs in single item and baske…

### PR DESCRIPTION
**Pull request by stentrop**
…t view

### All changes meet the following requirements
- [x] Changelog entry was added
- [ ] Changes have been documented
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer

@plentymarkets/plentyshop

### CODE REVIEW

**Comment by stentrop** [comment]
Bildschirmfoto 2021-12-06 um 08 58 53

File: resources/js/src/app/components/basket/list/BasketListItem.vue [hunk 1]
```diff
@@ -386,6 +390,17 @@ export default {
 
             return false;
         },
+        isPropertyWithAdditionalCostsVisible(propertyId)
+        {
+          const property = this.basketItem.variation.data.properties.find(property => property.property.id === parseInt(propertyId));
```
Comment (lks-mtzn) [inline resources/js/src/app/components/basket/list/BasketListItem.vue]: Hier suchst du die selbe Property aus der du die ID ausließt.

File: resources/js/src/app/components/item/ItemPrice.vue [hunk 2]
```diff
@@ -67,6 +78,21 @@ export default {
         }
     },
 
+    methods:
+   {
+     isPropertyWithAdditionalCostsVisible(propertyId)
+     {
+       const property = this.currentVariation.properties.find(property => property.property.id === parseInt(propertyId));
```
Comment (lks-mtzn) [inline resources/js/src/app/components/item/ItemPrice.vue]: Hier auch.

**Review by lks-mtzn — APPROVED** [review]

**Review by lks-mtzn — CHANGES_REQUESTED** [review]
.

**Review by Tim-M-S — DISMISSED** [review]

**Review by Tim-M-S — DISMISSED** [review]

File: resources/lang/de/Template.properties [hunk 3]
```diff
@@ -58,6 +58,7 @@ addressZip = "PLZ"
 ; address - Addressen
 ; basket - Warenkorb
 basket = "Warenkorb"
+basketAdditionalCosts = "Zusatzkosten (Preis pro Artikel)"
```
Comment (maxiroellplenty) [inline resources/lang/de/Template.properties:61]: Add missing translations for en/fr/nl/pl

**Review by maxiroellplenty — APPROVED** [review]

**Review by lks-mtzn — APPROVED** [review]

**Pull request merged**